### ISSUE TITLE
[Yume] Yume does not return a error code when there is a dependency issue

### ISSUE THREAD

**Issue by dikim33** [oscar-cluster/oscar#508]
**Reported by valleegr on 13 Jan 2009 20:04 UTC**
```
$ sudo yume --repo http://mirror.centos.org/centos/5/os/i386/ install php-devel
os_i386                   100% |=========================| 1.1 kB    00:00
Setting up Install Process
Parsing package install arguments
Resolving Dependencies
--> Running transaction check
---> Package php-devel.i386 0:5.1.6-20.el5 set to be updated
--> Processing Dependency: php = 5.1.6-20.el5 for package: php-devel
--> Finished Dependency Resolution
Error: Missing Dependency: php = 5.1.6-20.el5 is needed by package php-devel
[trunk](gvh@localhost)$ sudo vi /tftpboot/distro/centos-5-i386.url
[trunk](gvh@localhost)$ sudo yume --repo http://isoredirect.centos.org/centos/5/os/i386/ install php-devel
[trunk](gvh@localhost)$ sudo vi /tftpboot/distro/centos-5-i386.url
[trunk](gvh@localhost)$ sudo yume --repo http://mirror.centos.org/centos/5/os/i386/ install php-devel
os_i386                   100% |=========================| 1.1 kB    00:00
Setting up Install Process
Parsing package install arguments
Resolving Dependencies
--> Running transaction check
---> Package php-devel.i386 0:5.1.6-20.el5 set to be updated
--> Processing Dependency: php = 5.1.6-20.el5 for package: php-devel
--> Finished Dependency Resolution
Error: Missing Dependency: php = 5.1.6-20.el5 is needed by package php-devel

$ echo $?
0
```

**Comment by dikim33** [comment]
**Modified by valleegr on 14 Jan 2009 03:46 UTC**

**Comment by dikim33** [comment]
**Comment by valleegr on 19 Jan 2009 17:58 UTC**
(In [7856]) This check-in fixes #509 - Yume now catches errors from yum; ptty_try 
catches errors from yume, so we can have an overall error management, 
including dependencies errors.
